### PR TITLE
implement password reset

### PR DESCRIPTION
**Pull request by calexandar**
I think that token must be used for this operation.

### CODE REVIEW

**Comment by calexandar** [comment]
[Implement Password Reset](https://trello.com/c/BRHHZezr/25-implement-password-reset)

File: app/Http/Requests/Auth/ResetPasswordRequest.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'email' => 'required|email|exists:users,email',
```
Comment (davorminchorov) [inline app/Http/Requests/Auth/ResetPasswordRequest.php:25]: Please use the array syntax for all validation rules to keep it consistent.

File: app/Http/Controllers/Auth/ResetPasswordController.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    public function index() :Response
+    {
+              
+        return Inertia::render('Auth/ResetPassword');        
+    }
+
+    public function store(ResetPasswordRequest $request)
+    {
+        
+        $request->validated();
```
Comment (davorminchorov) [inline app/Http/Controllers/Auth/ResetPasswordController.php:24]: You are not using the result. You can remove it if you don't need it.

**Review by davorminchorov — CHANGES_REQUESTED** [review]

**Comment by davorminchorov** [comment]
You seem to be missing the whole workflow of resetting a password. The request new password change and the reset password workflows are both included in this ticket.

**Pull request closed without merging**